tests/foreman/cli/test_ansible.py

```diff
@@ -4,156 +4,544 @@
-        5. Run the Ansible playbook associated with that role
-        6. Check if the job is executed successfully.
-        7. Disassociate the Role from the host.
-        8. Delete the assigned ansible role
+        :id: 0c52bc63-a41a-4f48-a980-fe49b4ecdbdc
 
-    :expectedresults:
-        1. Host should be assigned the proper role.
-        2. Job execution must be successful.
-        3. Operations performed with hammer must be successful.
+        :steps:
+            1. Register a content host with satellite
+            2. Import a role into satellite
+            3. Assign that role to a host
+            4. Assert that the role and variable were assigned to the host successfully
+            5. Run the Ansible playbook associated with that role
+            6. Check if the job is executed successfully.
+            7. Disassociate the Role from the host.
+            8. Delete the assigned ansible role
 
-    :BZ: 2154184
+        :expectedresults:
+            1. Host should be assigned the proper role.
+            2. Job execution must be successful.
+            3. Operations performed with hammer must be successful.
 
-    :customerscenario: true
+        :BZ: 2154184
 
-    :CaseImportance: Critical
-    """
-    SELECTED_ROLE = 'RedHatInsights.insights-client'
-    SELECTED_ROLE_1 = 'theforeman.foreman_scap_client'
-    SELECTED_VAR = gen_string('alpha')
-    # disable batch tasks to test BZ#2154184
-    target_sat.cli.Settings.set({'name': 'foreman_tasks_proxy_batch_trigger', 'value': 'false'})
-    if rhel_contenthost.os_version.major <= 7:
-        rhel_contenthost.create_custom_repos(rhel7=settings.repos.rhel7_os)
-        assert rhel_contenthost.execute('yum install -y insights-client').status == 0
-    rhel_contenthost.install_katello_ca(target_sat)
-    rhel_contenthost.register_contenthost(module_org.label, force=True)
-    assert rhel_contenthost.subscribed
-    rhel_contenthost.add_rex_key(satellite=target_sat)
-    proxy_id = target_sat.nailgun_smart_proxy.id
-    target_host = rhel_contenthost.nailgun_host
-
-    target_sat.cli.Ansible.roles_sync(
-        {'role-names': f'{SELECTED_ROLE},{SELECTED_ROLE_1}', 'proxy-id': proxy_id}
-    )
+        :customerscenario: true
+        """
+        SELECTED_ROLE = 'RedHatInsights.insights-client'
+        SELECTED_ROLE_1 = 'theforeman.foreman_scap_client'
+        SELECTED_VAR = gen_string('alpha')
+        # disable batch tasks to test BZ#2154184
+        target_sat.cli.Settings.set({'name': 'foreman_tasks_proxy_batch_trigger', 'value': 'false'})
+        if rhel_contenthost.os_version.major <= 7:
+            rhel_contenthost.create_custom_repos(rhel7=settings.repos.rhel7_os)
+            assert rhel_contenthost.execute('yum install -y insights-client').status == 0
+        rhel_contenthost.install_katello_ca(target_sat)
+        rhel_contenthost.register_contenthost(module_org.label, force=True)
```

Comment:
`forece` by default set to `True`
```suggestion
        rhel_contenthost.register_contenthost(module_org.label)
```

Reply:
yeah, moved it to Global registration instead